### PR TITLE
Contributing

### LINKED ISSUE #13: Add Contribution Guidelines
The project needs to have a basic contribution guidelines.

### PR DESCRIPTION
**Pull request by ydjainopensource**
I have added contribution guidelines.....simply merged the guidelines from Clear containers and RunV repo.....I have also included few changes in the readme to point to the license and contact info.

Fixes #13

### CODE REVIEW

**Comment by jodh-intel** [comment]
Hi @ydjainopensource - thanks for raising this! I absolutely agree that we need a contributing guide however, I think we need to decide where they should live.

Since there are quite a few repos, and since we want to avoid having multiple copies of identical or near-identical docs (excepting the `LICENSE` files) I'm actually wondering if there should be:

- a *single* contributing guide (with actual content) that lives in https://github.com/kata-containers/community
- "stub" contributing guides in each of the other repos that basically say, "See https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md".

This approach would ease the maintenance a lot since otherwise we may end up having to apply changes to each individual `CONTRIBUTING.md` in all the different repos.

The other point is that we can't land some of these docs as we haven't yet applied the processes they are referring to. For example, we haven't yet agreed whether to adopt the Clear Containers https://github.com/clearcontainers/tests/tree/master/cmd/checkcommits tool that we use in that project to enforce the `Fixes #XXX` logic. Coincidentally, I raised a PR today to discuss exactly that topic:

- https://github.com/kata-containers/agent/pull/77

I've also raised a PR on the community repo to explain how to get started (as a user, not as a dev so we must do that somewhere soon):

- https://github.com/kata-containers/community/pull/3

In summary, I think that there is still a bit of discussion to be had on how we document contributions and what processes and practises we're going to pull in from the Clear Containers and runv projects.

This seems like as good a place as any to start those discussions so...

ping @sameo, @gnawux, @bergwolf, @laijs, @grahamwhaley, @rcaballeromx, @klynnrif.

**Comment by ydjainopensource** [comment]
Moved to new PR #15 as this one was failing

Sorry

**Comment by sameo** [comment]
> Since there are quite a few repos, and since we want to avoid having multiple copies of identical or near-identical docs (excepting the LICENSE files) I'm actually wondering if there should be:
> *  a single contributing guide (with actual content) that lives in https://github.com/kata-containers/community
> *  "stub" contributing guides in each of the other repos that basically say, "See https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md".

Yes, that would make our lives much easier.

> In summary, I think that there is still a bit of discussion to be had on how we document contributions and what processes and practises we're going to pull in from the Clear Containers and runv projects.

I think we agreed on a common set of processes, and we presented those to the community meetup at Kubecon (I'm trying to find the slides...). @gnawux @resouer @bergwolf Would you agree?

**Comment by jodh-intel** [comment]
Apologies @sameo - that was a bad turn of phrase on my part. I was just trying to suggest that we haven't yet agreed on some of the minutae of the processes that we *have* agreed upon (essentially how we apply the agreed processes). Things like `checkcommits`, particular github labels, some of the CI checks we use in Clear Containers, that type of thing. There was also mention of [cc-collect-data.sh](https://github.com/clearcontainers/runtime/blob/master/data/cc-collect-data.sh.in) in this PR which falls into the same category, but looking again I see that that has been commented out.
